020-Getting-started/050-sql-access.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: SQL access
+keywords: ['sql', 'access', 'rest']
+description: How to access Xata using SQL directly
+slug: getting-started/sql-access
+published: false
+---
```

Comment:
@philkra Given the way you've now set up the doc SDKs, should we not put this content into an `sdk/sql/overview` page? Feels a little out of place in getting started?

Then I think we probably need to mention SQL in the how Xata works / concepts sections?

Reply:
I am a plus one to move to the page into the SDK section. Currently, it's only one page for SQL, so a sub-folder structure is not necessary. Let me push a commit.

> Then I think we probably need to mention SQL in the how Xata works / concepts sections?

Fair, point but we can do that in a separate PR.